doc/build-unix.md

```diff
@@ -200,6 +200,40 @@ or building and depending on a local version of Berkeley DB 4.8. The readily ava
 As mentioned above, when maintaining portability of the wallet between the standard Gridcoin distributions and independently built
 node software is desired, Berkeley DB 4.8 must be used.
 
+Bulding on Raspbian/Raspberry Pi
+-------------------
+Building on a current [Raspbian Stretch Lite](https://www.raspberrypi.org/downloads/raspbian/):
+You should add a Swapfile to compensate the low memory on the device. I used 4 GB.
+
+Clone the Repository. Probably you can also use the Gridcoin-Community repo https://github.com/gridcoin-community/Gridcoin-Research
+
+```bash
+git clone https://github.com/gridcoin/Gridcoin-Research
+cd Gridcoin-Research
+```
+Get the dependencies. 
+```bash
+sudo apt-get install build-essential libtool autotools-dev automake pkg-config libssl-dev libevent-dev bsdmainutils libboost-system-dev libboost-filesystem-dev libboost-chrono-dev libboost-program-options-dev libboost-test-dev libboost-thread-dev libboost-all-dev
+```
+You will need BerkeleyDB 4.8 (see in the other guides), so you need to compile it yourself. You can use "make -j4" if you have 4 cores to increase the speed, otherwise scratch "-j4".
+```bash
+wget http://download.oracle.com/berkeley-db/db-4.8.30.NC.tar.gz
+tar -xzvf db-4.8.30.NC.tar.gz
+cd db-4.8.30.NC/build_unix
+../dist/configure --enable-cxx
+make -j4
+sudo make install
+export BDB_INCLUDE_PATH="/usr/local/BerkeleyDB.4.8/include"
+export BDB_LIB_PATH="/usr/local/BerkeleyDB.4.8/lib"
+ln -s /usr/local/BerkeleyDB.4.8/lib/libdb-4.8.so /usr/lib/libdb-4.8.so
+```
+Compile with the following commands. The ./configure flags are set your system that it should use less memory to compile than usual and that you have BerkeleyDB 4.8.
+```bash
+./autogen.sh
+ ./configure CXXFLAGS="--param ggc-min-expand=1 --param ggc-min-heapsize=32768" CPPFLAGS="-I/usr/local/BerkeleyDB.4.8/include -O2" LDFLAGS="-L/usr/local/BerkeleyDB.4.8/lib"
+make -j4
+sudo make install
```

Comment:
`sudo make install` should not be used on any linux system with package manager. It would make it hard to remove, or update, later. For debian `checkinstall`(...) should be used.

Reply:
@tomasbrod TIL :)

Reply:
I had to use sudo in my case I think :(

Reply:
would have to use sudo if compiling under a normal privileged account. if compiling as root you don't need sudo. least in ubuntu anyway

Reply:
The discussion is not about using sudo. Priviledge escalation should be used when writing to root. Checkinstall should be used over make install. `make install` is fine for some systems, but on Raspbian (which is Debian based), `checkinstall` should be used instead.

Reply:
so I could try "checkinstall" instead of "sudo make install" ? I will check and report back when I tried

Reply:
If you already run something in there with sudo, chances are it messed up permissions. Either delete the folder and start from scratch, or fix the permissions.

Reply:
@Quezacoatl1 Did you get a chance to try it out?